plugin/trino-druid/src/main/java/io/trino/plugin/druid/DruidJdbcClient.java

```diff
@@ -596,4 +596,29 @@ private WriteMapping legacyToWriteMapping(Type type)
         }
         throw new TrinoException(NOT_SUPPORTED, "Unsupported column type: " + type.getDisplayName());
     }
+
+    @Override
+    public RemoteIdentifiers getRemoteIdentifiers(Connection connection)
+    {
+        return new RemoteIdentifiers()
```

Comment:
How about reusing `super.getRemoteIdentifiers(connection)`?

Reply:
I need to override for Druid : 
```
            @Override
            public boolean shouldUpperCase()
            {
                return false;
            }
```

Reply:
refactored